deposit/src/main/java/edu/unc/lib/deposit/normalize/Proquest2N3BagJob.java

```diff
@@ -279,44 +266,39 @@ private Resource populateSimple(Model model, PID primaryPID, File contentFile) {
 		Resource primaryResource = model.createResource(primaryPID.getURI());
 
 		// use the filename as the label
-		model.add(primaryResource, dprop(model, label), contentFile.getName());
+		model.add(primaryResource, CdrDeposit.label, contentFile.getName());
 
 		// Reference the content file as the data file
-		model.add(primaryResource, dprop(model, stagingLocation), getRelativePath(contentFile));
+		model.add(primaryResource, CdrDeposit.stagingLocation, getRelativePath(contentFile));
 
 		return primaryResource;
 	}
 
 	private Resource populateAggregate(Model model, PID primaryPID, List<?> attachmentElements, File attachmentDir,
 			File contentFile, String title) {
 
-		Property labelP = dprop(model, label);
-		Property fileLocation = dprop(model, stagingLocation);
-		Property defaultWebObjectP = cdrprop(model, defaultWebObject);
-		Property hasModelP = fprop(model, hasModel);
-
 		// Create the primary resource as a bag
 		Bag primaryBag = model.createBag(primaryPID.getURI());
 
-		model.add(primaryBag, hasModelP, model.createResource(CONTAINER.getURI().toString()));
-		model.add(primaryBag, hasModelP, model.createResource(AGGREGATE_WORK.getURI().toString()));
+		model.add(primaryBag, RDF.type, model.createResource(CONTAINER.getURI().toString()));
```

Comment:
Not seeing this one in my local code. Are you sure this hasn't been changed already?